src/components/calculator/Button.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+function Button(props) {
+    const { onClick, value } = props;
+
+    if (value === 0) {
+        return (
+            <div className="button  wide"><button onClick={onClick} value={value}>{value}</button></div>
+        )
+    } else {
+        return (
+            <div className={value === "x" || value === "-" || value === "+" || value === "÷" || value === "=" ? "button orange" : "button"}>
+                <button onClick={onClick} value={value}>{value}</button>
```

Comment:
What's the meaning of this value prop in the button component ?

Reply:
I need that to use on handleOperator function in order to know which operator was clicked.